### PR TITLE
Patch for events order in rest api

### PR DESCRIPTION
**Pull request by Mte90**
The url now is (this two are the same):

* http://192.168.1.10:8000/api/remo/v1/events/?format=json&orderby=DESC
* http://192.168.1.10:8000/api/remo/v1/events/?format=json

This order by ASC:
* http://192.168.1.10:8000/api/remo/v1/events/?format=json&orderby=ASC

### CODE REVIEW

**Comment by MichaelKohler** [comment]
I'll test this tonight.

File: remo/events/api/views.py [hunk 1]
```diff
@@ -43,6 +43,13 @@ def retrieve(self, request, pk):
                                              context={'request': request})
         return Response(serializer.data)
 
+    def get_queryset(self):
+        orderby = self.request.query_params.get('orderby', 'DESC')
+
+        if orderby != 'DESC':
```
Comment (MichaelKohler) [inline remo/events/api/views.py]: This check is the wrong way around. We want the latest events shown on the first page by default (and also when using DESC actually). So this should be ```orderby  == 'DESC'```. See https://github.com/mozilla/remo/pull/1407 (which originally broke Community Analytics in the first place). For Community Analytics they will need to use the ```orderby=ASC``` filter in their configuration.

Default & DESC:
* Event 2018-02-28
* Event 2018-01-15
* Event 2017-11-15

ASC:
* Event 2017-11-15
* Event 2018-01-15
* Event 2018-02-28

Would you mind changing this?

Comment (Mte90) [reply remo/events/api/views.py]: Uhm you are right is like the opposite right now.
I confused the order of the demo events.

**Comment by Mte90** [comment]
Changed as requested.

**Comment by MichaelKohler** [comment]
Looks good to me now, but didn't test it locally. @akatsoulas can we get this merged and deployed? Then Community Analytics can use the ASC order in their script and can fetch the missing Reps data again.

**Comment by Mte90** [comment]
Waiting the push on production to ping the community-analytics about it :-)

**Comment by akatsoulas** [comment]
This is now released in prod.

**Pull request merged**